src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/UpToDate/BuildUpToDateCheck.cs

```diff
@@ -75,27 +80,14 @@ internal sealed partial class BuildUpToDateCheck : OnceInitializedOnceDisposedUn
             _projectItemSchemaService = projectItemSchemaService;
             _telemetryService = telemetryService;
             _fileSystem = fileSystem;
-        }
-
-        public Task LoadAsync()
-        {
-            return InitializeAsync();
-        }
 
-        public Task UnloadAsync()
-        {
-            return ExecuteUnderLockAsync(_ =>
-            {
-                lock (_stateLock)
+            threadingService.RunAndForget(
+                async () =>
```

Comment:
There's no guarantee that the MEF component represented by configuredProjectActivationTracking is valid at the point that this is executed.